toolchain/semantics/semantics_ir.h

```diff
@@ -261,10 +262,12 @@ class SemanticsIR {
   }
 
   // Produces a string version of a type.
```

Comment:
Add a comment to document how `in_type_context` impacts this?

Reply:
Done.